_events/2023-11-08-intro-workshop.md

```diff
@@ -0,0 +1,60 @@
+---
+short_title: CHTC Introduction Workshop
+title: "CHTC Introduction Workshop"
```

Comment:
```suggestion
title: "Hands-on Introduction to CHTC Resources Workshop"
```

Reply:
I'd consider re-titling this workshop just because "CHTC Introduction workshop" and "CHTC Information Session" are similar phrases for someone quickly skimming.